usr.sbin/daemon/daemon.c

```diff
@@ -613,6 +577,7 @@ restrict_process(const char *user)
  * Return value of false is assumed to mean EOF or error, and true indicates to
  * continue reading.
  */
+// TODO: simplify signature - state contains pipefd
```

Comment:
Wrong comment style, if it's going to stay. Ideally just merge it into the block just above with a blank line separating.

Reply:
fixed